patches/api/0484-add-ItemFrameMeta-API.patch

```diff
@@ -0,0 +1,88 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TehBrian <tehbrian@proton.me>
+Date: Thu, 15 Aug 2024 17:06:29 -0400
+Subject: [PATCH] add ItemFrameMeta API
+
+
+diff --git a/src/main/java/io/papermc/paper/inventory/meta/ItemFrameMeta.java b/src/main/java/io/papermc/paper/inventory/meta/ItemFrameMeta.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..65b388e0a48a5927448a79b0f2e7742895038034
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/inventory/meta/ItemFrameMeta.java
+@@ -0,0 +1,53 @@
++package io.papermc.paper.inventory.meta;
++
++import org.bukkit.inventory.meta.ItemMeta;
++
++public interface ItemFrameMeta extends ItemMeta {
```

Comment:
Are we making the same mistake here lots of other ItemMetas have done? technically there are 3 states for every boolean. It could be true, false, or not present in the custom data.